roles/nvidia-gpu-operator/defaults/main.yml

```diff
@@ -19,7 +19,17 @@ gpu_operator_namespace: "gpu-operator-resources"
 gpu_operator_default_runtime: "docker"
 gpu_operator_driver_registry: "nvcr.io/nvidia"
 gpu_operator_driver_version: "460.32.03"
-gpu_operator_plugin_args: "--mig-strategy={{ k8s_gpu_mig_strategy }},--pass-device-specs=true,--fail-on-init-error=true,--device-list-strategy=envvar,--nvidia-driver-root=/run/nvidia/driver"
+deepops_dir: "/opt/deepops"
+gpu_operator_config_dir: "{{ deepops_dir }}/gpu-operator"
+gpu_operator_plugin_args:
+- "--mig-strategy={{ k8s_gpu_mig_strategy }}"
+- "--pass-device-specs=true"
+- "--fail-on-init-error=true"
+- "--device-list-strategy=envvar"
+- "--nvidia-driver-root=/run/nvidia/driver"
+gpu_operator_dcgm_version: "2.1.8-2.4.0-rc.2-ubuntu20.04"
```

Comment:
We need to remember to upload this var (and all the above defaults) every time we bump the version of the GPU Operator.

I have been trying to keep our defaults in-sync with the vars in the GPU Operator values.yml.